scenarios/telemetry/dotnet/telemetry_consumer/Worker.cs

```diff
@@ -14,36 +13,44 @@ public class Worker : BackgroundService
     public Worker(ILogger<Worker> logger, IConfiguration configuration)
     {
         _logger = logger;
-        _configuration  = configuration;
+        _configuration = configuration;
     }
 
     protected override async Task ExecuteAsync(CancellationToken stoppingToken)
     {
         var cs = MqttConnectionSettings.CreateFromEnvVars(_configuration.GetValue<string>("envFile")!);
         _logger.LogInformation("Connecting to {cs}", cs);
 
-        var mqttClient = new MqttFactory().CreateManagedMqttClient(MqttNetTraceLogger.CreateTraceLogger());
+        var mqttClient = new MqttFactory().CreateMqttClient(MqttNetTraceLogger.CreateTraceLogger());
 
-        mqttClient.InternalClient.ConnectedAsync += async cea =>
+        mqttClient.DisconnectedAsync += async d =>
```

Comment:
Must this delegate be async? Is there a benefit to calling `await Task.Yield()`?

Reply:
we can define these delegates w/o async and return `Task.CompletedTask`